src/SwedbankPay.Sdk/Consumers/ConsumersResponse.cs

```diff
@@ -1,29 +1,26 @@
-﻿using Newtonsoft.Json;
-
-namespace SwedbankPay.Sdk.Consumers
-{
-    public class ConsumersResponse
-    {
-        [JsonConstructor]
-        public ConsumersResponse(string token)
-        {
-            Token = token;
-        }
-
-
-        public ConsumersResponse()
-        {
-        }
-
-
-        /// <summary>
-        ///     The array of operation objects to choose from
-        /// </summary>
-        public OperationList Operations { get; protected set; } = new OperationList();
-
-        /// <summary>
-        ///     A session token used to initiate Checkout UI
-        /// </summary>
-        public string Token { get; }
-    }
+﻿namespace SwedbankPay.Sdk.Consumers
+{
+    public class ConsumersResponse
+    {
+        public ConsumersResponse(string token)
+        {
+            Token = token;
+        }
+
+
+        public ConsumersResponse()
+        {
+        }
+
+
+        /// <summary>
+        ///     The array of operation objects to choose from
+        /// </summary>
+        public IOperationList Operations;

```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)